app/repositories/ArrivalMovementRepository.scala

```diff
@@ -231,8 +243,19 @@ class ArrivalMovementRepository @Inject()(mongo: ReactiveMongoApi)(implicit ec:
       )
 
     collection.flatMap {
-      _.findAndUpdate(selector, modifier)
-        .map {
+      _.findAndUpdate[JsObject, JsObject](
+        selector = selector,
+        update = modifier,
+        fetchNewObject = true,
+        upsert = true,
```

Comment:
Same here, upsert should be false

Reply:
it was true earlier,just used what ever was set